pyproject.toml

```diff
@@ -1,10 +1,10 @@
 [build-system]
-requires = ["hatchling", "hatch-vcs", "hatch-requirements-txt"]
+requires = ["hatchling", "uv-dynamic-versioning"]
 build-backend = "hatchling.build"
 
 [project]
 name = "nsls2api"
-dynamic = ["version", "dependencies"]
+dynamic = ["version"]
```

Comment:
Was it intentional to remove dynamic dependencies?
https://github.com/ninoseki/uv-dynamic-versioning/blob/main/docs/metadata_hook.md